felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/Tekst.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.helsearbeidsgiver.felles
+
+object Tekst {
```

Comment:
hva med FeilTekst?

Reply:
Jeg tenkte i utgangspunktet at det kan brukes som et samlingspunkt for brukerrettede tekster, og det kan også være tekster som ikke har noe med feil å gjøre. Jeg lar det være inntil videre, så kan vi heller endre det dersom det kommer flere tekster.